### PR TITLE
Move drop down to header

### PR DESCRIPTION
**Pull request by jordan621k**

### CODE REVIEW

File: src/component/Form.js [hunk 1]
```diff
@@ -108,31 +108,29 @@ class Form extends React.Component {
...
-      value: i18n(this.context.locale).dropDownOptions.language[this.context.locale]
-    }
-  }
+  // getLanguageDropDownProps () {
+  //   return {
+  //     name: 'languageDropDownMenu',
+  //     options: i18n(this.context.locale).dropDownOptions.language,
+  //     value: i18n(this.context.locale).dropDownOptions.language[this.context.locale]
+  //   }
+  // }
```
Comment (x955199) [inline src/component/Form.js]: 這是怎樣啦？

File: src/component/Form.js [hunk 2]
```diff
@@ -108,31 +108,29 @@ class Form extends React.Component {
...
-                <DropDownMenu {...this.getCountryDropDownProps()}/>
-                &nbsp; {i18n(this.context.locale).form.in} &nbsp;
-                <DropDownMenu {...this.getLanguageDropDownProps()} callback={updateLocale}/>
-              </b>
-              <br/>
-            </form>
-          )}
-        </LocaleContext.Consumer>
+        {/* <LocaleContext.Consumer> */}
+        {/* {({ locale, updateLocale }) => ( */}
```
Comment (x955199) [inline src/component/Form.js]: 這是怎樣啦？

File: src/component/Form.js [hunk 3]
```diff
@@ -108,31 +108,29 @@ class Form extends React.Component {
...
+        <form className="Form" onSubmit={this.handleSubmit}>
+          <b>
+            <DropDownMenu {...this.getCategoryDropDownProps()}/>
+            &nbsp; {i18n(this.context.locale).form.from} &nbsp;
+            <DropDownMenu {...this.getCountryDropDownProps()}/>
+          </b>
+          <br/>
+        </form>
+        {/* )} */}
+        {/* </LocaleContext.Consumer> */}
```
Comment (x955199) [inline src/component/Form.js]: 這是怎樣啦？

**Comment by x955199** [comment]
.............

File: src/component/Header.js [hunk 4]
```diff
@@ -1,15 +1,44 @@
...
   render () {
     return (
-      <header className="Header">
-        <img src={logo} alt="logo" />
-        <h6>{i18n(this.context.locale).header}</h6>
-      </header>
+      <LocaleContext.Consumer>
+        {({ locale, updateLocale }) => (
+          <header className="Header">
+            <div className="logoAndText">
```
Comment (jordan621k) [inline src/component/Header.js]: change name

File: src/component/Header.js [hunk 5]
```diff
@@ -1,15 +1,44 @@
...
+              <img className="logoImage" src={logo} alt="logo" />
+            </div>
+            <div className="formBlock">
```
Comment (jordan621k) [inline src/component/Header.js]: change name

File: src/component/Header.js [hunk 6]
```diff
@@ -1,15 +1,44 @@
...
+              <h6>{i18n(this.context.locale).header}</h6>
+            </div>
+            <div className="languageBlock">
```
Comment (jordan621k) [inline src/component/Header.js]: language dropdown

File: src/component/Header.js [hunk 7]
```diff
@@ -1,15 +1,44 @@
...
 import logo from '../static/logo.png'
 import { i18n, LocaleContext } from '../i18n/i18n'
+import Form from './Form'
+import { Router } from 'react-router'
+import { createBrowserHistory } from 'history'
+import DropDownMenu from './DropDownMenu'
 
 class Header extends React.Component {
+
+  constructor (props) {
```
Comment (jordan621k) [inline src/component/Header.js]: delete

File: src/component/Video.js [hunk 8]
```diff
@@ -1,25 +1,44 @@
...
+  onMouseOverFunction (e) {
+    this.changeMouseCursor(e)
+    this.changeBoarder(e)
+  }
+
   changeMouseCursor (e) {
     e.target.style.cursor = 'pointer'
   }
 
+  changeBoarder (e) {
```
Comment (jordan621k) [inline src/component/Video.js]: addBoarder

File: src/component/Video.js [hunk 9]
```diff
@@ -1,25 +1,44 @@
...
+    if (e.target.className === 'Video') {
+      e.target.style.border = '#66fcf1 2px solid'
+    }
+  }
+
+  unchangeBoarder (e) {
```
Comment (jordan621k) [inline src/component/Video.js]: removeBorder

File: src/component/Video.js [hunk 10]
```diff
@@ -1,25 +1,44 @@
...
     window.open(this.props.videoLink)
   }
 
+  onMouseOverFunction (e) {
+    this.changeMouseCursor(e)
+    this.changeBoarder(e)
+  }
+
   changeMouseCursor (e) {
     e.target.style.cursor = 'pointer'
```
Comment (jordan621k) [inline src/component/Video.js]: change to add class,

File: src/component/Video.js [hunk 11]
```diff
@@ -1,25 +1,43 @@
...
-  changeMouseCursor (e) {
-    e.target.style.cursor = 'pointer'
+  addBorderAndMouseCursor (e) {
+    if (e.target.className === 'Video') {
+      e.target.classList.add('selected')
+    }
+  }
+
+  removeBorder (e) {
+    if (document.querySelectorAll('div.Video.selected')[0]) {
```
Comment (jordan621k) [inline src/component/Video.js]: add var

File: src/component/Video.js [hunk 12]
```diff
@@ -1,25 +1,43 @@
...
+    this.removeBorder = this.removeBorder.bind(this)
   }
 
   openVideoInWindow () {
     window.open(this.props.videoLink)
   }
 
-  changeMouseCursor (e) {
-    e.target.style.cursor = 'pointer'
+  addBorderAndMouseCursor (e) {
```
Comment (jordan621k) [inline src/component/Video.js]: change name

File: src/component/Video.js [hunk 13]
```diff
@@ -1,25 +1,43 @@
...
+    }
+  }
+
+  removeBorder (e) {
+    if (document.querySelectorAll('div.Video.selected')[0]) {
+      document.querySelectorAll('div.Video.selected')[0].classList.remove('selected')
+    }
+  }
+
+  showUploadInfo () {
```
Comment (jordan621k) [inline src/component/Video.js]: getUploadInfo

File: src/css/App.css [hunk 14]
```diff
@@ -15,4 +15,12 @@ body {
 
 html {
   height: 100%;
+}
+
+.Form {
```
Comment (jordan621k) [inline src/css/App.css:20]: form

File: src/component/Video.js (hunk 12 above)
Comment (jordan621k) [reply src/component/Video.js]: addMouseOverClass

**Pull request merged**